packages/vm/tests/api/index.spec.ts

```diff
@@ -1,20 +1,19 @@
-import * as tape from 'tape'
-import { Account, Address, KECCAK256_RLP } from '@ethereumjs/util'
+// explicitly import util and buffer,
+// needed for karma-typescript bundling
+import { Buffer } from 'buffer' // eslint-disable-line @typescript-eslint/no-unused-vars
```

Comment:
I think the eslint-disable might not be necessary for Buffer in this instance, as Buffer is used in the file.